src/app/zap-templates/zcl/data-model/chip/service-area-cluster.xml

```diff
@@ -32,11 +32,11 @@ limitations under the License.
     <item name="Name"    type="char_string" length="64" optional="false"/>
   </struct>
 
-  <struct name="LocationStruct">
+  <struct name="AreaStruct">
     <cluster code="0x0150"/>
-    <item name="LocationID"   type="int32u"                  optional="false" isNullable="false"/>
-    <item name="MapID"        type="int8u"                   optional="false" isNullable="true"/>
-    <item name="LocationInfo" type="LocationInfoStruct"      optional="false" isNullable="false"/>
+    <item name="AreaID"   type="int32u"              optional="false" isNullable="false"/>
+    <item name="MapID"    type="int8u"               optional="false" isNullable="true"/>
+    <item name="AreaDesc" type="AreaInfoStruct"      optional="false" isNullable="false"/>
```

Comment:
Spec feedback: This should perhaps be AreaDescriptor or AreaInfo naming-wise?

Reply:
I'm resolving this for now. I have brought this to the attention of @plauric to consider any spec changes.